Lesson6.ipynb

```diff
@@ -27,7 +34,686 @@
+    "#Start here\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What Is Object-Oriented Programming?\n",
+    "\n",
+    "So far, We have learned basic fundamentals in Python language. Now, let's focus on how to build a house from what we have learned. \n",
+    "\n",
+    "Object-oriented programming (OOP) is a programming model based on the concept of \"objects\", which may contain data, in the form of attributes and their behavior in the form of methods. \n",
+    "\n",
+    "A feature of objects is that an object's methods can access and often modify the data fields of the object with which they are associated (objects have a notion of \"this\" or \"self\"). In OOP, programs are designed by making them out of objects that interact with one another.\n",
+    "\n",
+    "Each Object has its own attributes, and behavior. Objects are separate from one another. They have their own existence, their own identity that is independent of other objects.\n",
+    "\n",
+    "For example, if you consider the Biggest Loser challenge, each member can be an object with a name, age, gender as its attributes and whether active participant or not can be its behavior.\n",
+    "\n",
+    "### Well, how do we construct this objects in our program? Class!\n",
+    "\n",
+    "### What is Class?\n",
+    "Well, a class is a place where you can identify the behaviors and properties of an Object. So, the properties and behavior of an Object will be defined inside a class.\n",
+    "\n",
+    "The syntax of the class definition:\n",
+    "\n",
+    "```python\n",
+    "\tclass ClassName:\n",
+    "\t\t'Optional class documentation string'\n",
+    "\t\tclass_suite\n",
+    "```\n",
+    "> *The class has a documentation string referred to a class variable ```__doc__```, which can be accessed via ```ClassName.__doc__```*\n",
+    "\n",
+    "> *The class_suite consists of all the component statements defining class members, data attributes, and functions.*\n",
+    "\n",
+    "Let's try it!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Member:\n",
+    "    \"Its a member class\"        #documentation string\n",
+    "    size = 0                    #Class variable\n",
+    "    def __init__(self, name, age, isCaptain, totalStepCount):    #initialize method called when class object is created\n",
+    "        self.name = name\n",
+    "        self.age = age\n",
+    "        self.isCaptain = \"Yes\" if isCaptain else \"No\"       #assign \"Yes\" if the isCaptian value is True else assigns \"No\"\n",
+    "        self.totalStepCount = totalStepCount\n",
+    "        Member.totalMembers += 1                            #totalMembers keeps increases by 1, everytime a new Member object is created"
```

Comment:
"totalMembers keeps increases by 1, everytime a new Member object is created"
Should be read
"totalMembers is increased by 1 every time a new instance of Member is created"